### PR TITLE
Add greeting to new members of the Slack

### PR DESCRIPTION
**Pull request by paked**
This PR adds a greeting to leaders who fill out our intake form.

Tests:

- Hackbot only sends out a message the first time the intake form is filled out
- Hackbot sends the right copy

### CODE REVIEW

**Comment by paked** [comment]
@MaxWofford this PR is ready to review. Since speaking to @kyleemile the copy has been redone, and we've agreed on how to integrate it properly into the onboarding process (how it is right now).

File: app/controllers/v1/leaders_controller.rb [hunk 1]
```diff
@@ -45,5 +47,40 @@ def welcome_letter_for_leader(leader)
         address: leader.address
       )
     end
+
+    def welcome_message(leader)
+      token = access_token leader
+      return if token.nil?
```
Comment (MaxWofford) [inline app/controllers/v1/leaders_controller.rb]: Do we want this to fail silently?

Comment (paked) [inline app/controllers/v1/leaders_controller.rb]: That's what we're doing elsewhere.

https://github.com/hackclub/api/blob/master/app/models/leader.rb#L84

File: app/controllers/v1/leaders_controller.rb [hunk 2]
```diff
@@ -45,5 +47,40 @@ def welcome_letter_for_leader(leader)
...
+
+      im = SlackClient::Chat.open_im(leader.slack_id, token)
+      return if !im[:ok] || !im[:latest].nil?
```
Comment (paked) [inline app/controllers/v1/leaders_controller.rb]: If you mean:

`return unless im[:ok] OR !im[:latest].nil?`

Then yes.

**Comment by paked** [comment]
@MaxWofford: Ready for another review! Sorry for the delay.

File: lib/data/copy/leaders_controller.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+welcome: >
```
Comment (MaxWofford) [inline lib/data/copy/leaders_controller.yml]: Let's change this to `welcome: |`. 
See http://stackoverflow.com/a/21699210/4018167.

**Comment by MaxWofford** [comment]
@paked Thoughts on formatting this like the check in code by having it schedule a job, then breaking this out into a "leader_welcome" conversation?

**Comment by paked** [comment]
@MaxWofford: It's an option. Just worried about overcomplicating this.

Probably good to separate the concerns.

**Pull request closed without merging**